lizrd/support/sync_and_version.py

```diff
@@ -76,11 +76,21 @@ def rsync_to_remote(host, local_dir):
         raise Exception(f"[RSYNC ERROR]: An error occurred during rsync: {str(e)}")
 
 
+def connection_to_athena_nick(connection):
+    d = {
+        "plglizard": "jaszczur",
+        "plgcrewtool": "crewtool",
+        "plgmaciejpioro": "maciejpioro",
+        "plgludziej": "ludziej",
+        "plgsimontwice": "simontwice",
+    }
+    user = connection.user
+    return d[user] if user in d else user
```

Comment:
connection.user[3:] maybe, or just `return d[user]` to raise error if user not specified?